src/routes/dispense.rs

```diff
@@ -346,14 +336,24 @@ pub async fn dispense_tokens(
 
     dispense_tracker.lock().unwrap().track(address);
 
+    clerk
```

Comment:
Maybe we want to teach `DispenseTracker` to work with `user_id` instead of addresses

Reply:
Does that mean we want to allow dispensing to the same address as long as it's requested via a different auth?